metadata.rb

```diff
@@ -2101,6 +2102,11 @@
           :description => "Given name field of ldap 'givenName' (default)",
           :type => 'string'
 
+
+#
+# Airflow settings
+#
+
 attribute "ldap/user_surname",
```

Comment:
Airflow attributes should go in the airflow metadata.rb. In any case please not in the middle of the ldap attributes.